src/lib/menu/menu-trigger.ts

```diff
@@ -104,6 +107,14 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
       this._createOverlay();
       this._overlayRef.attach(this._portal);
       this._subscribeToBackdrop();
+
+      this._globalKeydownHandler = this._renderer.listen('document', 'keydown',
```

Comment:
Hmm, it'll close both the menu and the dialog. What about focusing the menu panel, instead of the first option, when the menu is opened by mouse?